composer.json

```diff
@@ -29,6 +29,7 @@
         "drupal/paragraphs": "^1.13",
         "drupal/typed_link": "^2.0",
         "drush/drush": "^12",
+        "openeuropa/cdt-client": "dev-D8AGE-289",
```

Comment:
I added this as a repository/branch temporarily, until it is merged and put on packagist.org.
It is in `require-dev`, the same as `epoetry-client`.